diff --git a/node_modules/@radix-ui/react-context/dist/index.js b/node_modules/@radix-ui/react-context/dist/index.js
index 1f86fed..a742e49 100644
--- a/node_modules/@radix-ui/react-context/dist/index.js
+++ b/node_modules/@radix-ui/react-context/dist/index.js
@@ -37,7 +37,7 @@ module.exports = __toCommonJS(src_exports);
 
 // packages/react/context/src/createContext.tsx
 var React = __toESM(require("react"));
-var import_jsx_runtime = require("react/jsx-runtime");
+var import_jsx_runtime = require("react/jsx-runtime.js");
 function createContext2(rootComponentName, defaultContext) {
   const Context = React.createContext(defaultContext);
   function Provider(props) {
diff --git a/node_modules/@radix-ui/react-context/dist/index.mjs b/node_modules/@radix-ui/react-context/dist/index.mjs
index ab88942..1f1ee86 100644
--- a/node_modules/@radix-ui/react-context/dist/index.mjs
+++ b/node_modules/@radix-ui/react-context/dist/index.mjs
@@ -1,6 +1,6 @@
 // packages/react/context/src/createContext.tsx
 import * as React from "react";
-import { jsx } from "react/jsx-runtime";
+import { jsx } from "react/jsx-runtime.js";
 function createContext2(rootComponentName, defaultContext) {
   const Context = React.createContext(defaultContext);
   function Provider(props) {
