diff --git a/node_modules/@radix-ui/react-slot/dist/index.js b/node_modules/@radix-ui/react-slot/dist/index.js
index 724cd6f..737dcf0 100644
--- a/node_modules/@radix-ui/react-slot/dist/index.js
+++ b/node_modules/@radix-ui/react-slot/dist/index.js
@@ -39,7 +39,7 @@ module.exports = __toCommonJS(src_exports);
 // packages/react/slot/src/Slot.tsx
 var React = __toESM(require("react"));
 var import_react_compose_refs = require("@radix-ui/react-compose-refs");
-var import_jsx_runtime = require("react/jsx-runtime");
+var import_jsx_runtime = require("react/jsx-runtime.js");
 var Slot = React.forwardRef((props, forwardedRef) => {
   const { children, ...slotProps } = props;
   const childrenArray = React.Children.toArray(children);
diff --git a/node_modules/@radix-ui/react-slot/dist/index.mjs b/node_modules/@radix-ui/react-slot/dist/index.mjs
index 73e28af..dd31849 100644
--- a/node_modules/@radix-ui/react-slot/dist/index.mjs
+++ b/node_modules/@radix-ui/react-slot/dist/index.mjs
@@ -1,7 +1,7 @@
 // packages/react/slot/src/Slot.tsx
 import * as React from "react";
 import { composeRefs } from "@radix-ui/react-compose-refs";
-import { Fragment, jsx } from "react/jsx-runtime";
+import { Fragment, jsx } from "react/jsx-runtime.js";
 var Slot = React.forwardRef((props, forwardedRef) => {
   const { children, ...slotProps } = props;
   const childrenArray = React.Children.toArray(children);
